Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -79,7 +79,7 @@ equivalence: `LaurentSeries.LaurentSeriesRingEquiv` is the *topological, ring eq
 `K⸨X⸩ ≃+* RatFuncAdicCompl K`.
 * In order to compare `K⟦X⟧` with the valuation subring in the `X`-adic completion of
 `RatFunc K` we consider its alias `LaurentSeries.powerSeries_as_subring` as a subring of `K⸨X⸩`,
-that is itself clearly isomorphic (via `LaurentSeries.powerSeriesEquivSubring.symm`) to `K⟦X⟧`.
+that is itself clearly isomorphic (via `LaurentSeries.powerSeriesEquivSubring`) to `K⟦X⟧`.
```

Comment:
```suggestion
that is itself clearly isomorphic (via the inverse of `LaurentSeries.powerSeriesEquivSubring`) to `K⟦X⟧`.
```
What about this spelling? I 100% agree with having the link clickable, but `a` isomorphic to `b` via `f` seems to imply that `f : a ≃ b`, not `f : b ≃ a`. Anyway not very important.

Reply:
Alternatively, the `.symm` can go outside the backtick, as in `` `LaurentSeries.powerSeriesEquivSubring`.symm ``.